requirements.yml

```diff
@@ -45,7 +45,7 @@ collections:
     version: 2.1.0
   - name: https://github.com/stackhpc/ansible-collection-terraform
```

Comment:
This needs to change to `azimuth-cloud`, as do any uses of the roles from the collection.

Reply:
I'm actually surprised it works as it is? Probably the CI doesn't test the CaaS Terraform?

Reply:
Yeah CI doesn't test this, I just ran into this while testing manually in a dev azimuth :flushed: